pyodide-build/pyodide_build/common.py

```diff
@@ -269,6 +269,41 @@ def _get_sha256_checksum(archive: Path) -> str:
     return h.hexdigest()
 
 
+def ensure_wheel_platform(path: Path) -> Path:
```

Comment:
Are you sure this leaves universal wheels unchanged? Can you add a test for it? I think if  in a test, you give it an empty zipfile with a wheel-looking name it [should work](https://github.com/pypa/wheel/blob/e76040b0d970a0fe6211c31888fc929078ba02d9/src/wheel/cli/tags.py#L118)

Reply:
Universal wheels should be protected by the guard at the beginning:
```py
    if "emscripten" not in path.name:
        return path
```